build/docker/intel-idxd-config-initcontainer.Dockerfile

```
@@ -14,12 +14,22 @@
 ## See the License for the specific language governing permissions and
 ## limitations under the License.
 ###
-FROM debian:unstable-slim
-COPY ./LICENSE /licenses/intel-device-plugins-for-kubernetes/LICENSE
+FROM debian:stable-slim as build
 RUN apt-get update && apt-get install -y --no-install-recommends accel-config jq && rm -rf /var/lib/apt/lists/\*
+RUN mkdir -p /idxd-init/scratch
+ARG LIBS="libjson libaccel libjq libtinfo libonig libacl libselinux libpcre2-8"
+ARG EXECS="bash env accel-config jq sed grep"
+RUN mkdir /tmp/libs && for l in ${LIBS}; do cp "/lib/x86_64-linux-gnu/${l}"* /tmp/libs/; done
+RUN mkdir /tmp/bins && for b in ls ${EXECS}; do cp "/usr/bin/${b}"* /tmp/bins/; done
```

```suggestion
RUN mkdir /tmp/bins && for b in ${EXECS}; do cp "/usr/bin/${b}"* /tmp/bins/; done
```
Remove `ls` as it is not needed.